proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,74 @@
+# Proposal: i18n.getOSLanguage>
+
+**Summary**
+
+Allow to get the language of your operating system as an `IETF tag`
+
+**Document Metadata**
+
+**Author:** carlosjeurissen
```

Comment:
optional: maybe link to your github?  (Though I imagine you have carlosjeurissen just about everywhere : ))

Reply:
Added.